RadixWallet/Features/DappInteractionFeature/Coordinator/DappInteractionFlow.swift

```diff
@@ -1041,8 +1041,9 @@ extension DappInteractionFlow.Path.State {
 				nonce: .secureRandom(),
 				signTransactionPurpose: .manifestFromDapp,
 				message: item.message.map {
-					Message.plainText(value: .init(mimeType: "text", message: .str(value: $0)))
-				} ?? .none,
+//					Message.plainText(value: $0)
+					Message.plainText(plaintext: .init(mimeType: "application/text", message: .stringMessage(string: $0)))
```

Comment:
REVERT / FIXME add constructor in Sargon so that we do not need to specify mime type here!